Pipeline/pipeline.py

```diff
@@ -173,36 +191,34 @@ def feature_reduction(self, n):
 
         return self.X
 
-    def data_scaling(self, test_size, scaling=True, scalers=None):
+    def data_scaling(self, test_size, scaling=True, scalers=[]):

```

Comment:
It is not recommended using a list as a default parameter.

Reply:
Done!